docs/platforms/javascript/common/opentelemetry/using-opentelemetry-apis.mdx

```diff
@@ -65,9 +65,9 @@ tracer.startActiveSpan("span name", () => {
 
 You can also use any other tracer. All OpenTelemetry spans will be picked up by Sentry automatically.
 
-## Modifying the default OpenTelemetry TracerProvider
+## Exporting Telemetry Data with the default OpenTelemetry TracerProvider
```

Comment:
I think I would leave the previous heading, as it is more generic. and then just add more keywords/details below this? 🤔